WordPress/src/main/java/org/wordpress/android/ui/CommentFullScreenDialogFragment.kt

```diff
@@ -55,11 +59,30 @@ class CommentFullScreenDialogFragment : Fragment(), CollapseFullScreenDialogCont
             reply.setText(it.getString(EXTRA_REPLY))
             reply.setSelection(it.getInt(EXTRA_SELECTION_START), it.getInt(EXTRA_SELECTION_END))
             viewModel.init()
+
+            setupSuggestionServiceAndAdapter(it.getSerializable(EXTRA_SITE_MODEL) as SiteModel, reply)
         }
 
         return layout
     }
 
+    private fun setupSuggestionServiceAndAdapter(mSite: SiteModel, reply: SuggestionAutoCompleteText) {
+        if (!isAdded || mSite == null || !SiteUtils.isAccessedViaWPComRest(mSite)) {
+            return
+        }
+        val mSuggestionServiceConnectionManager = SuggestionServiceConnectionManager(
```

Comment:
The `m` prefix is only used for Java instance variables. For Kotlin, we don't use it. 🙂